lib/rainforest/cli/http_client.rb

```diff
@@ -33,8 +33,8 @@ def post(url, body = {})
       JSON.parse(response.body)
     end
 
-    def get(url, body = {}, retries_on_failures: false)
-      wrap_exceptions(retries_on_failures) do
+    def get(url, body = {}, options = { retries_on_failures: false })
```

Comment:
I'd actually make `options` default to `{}` ("default options hash" has different semantics from "default arguments" and can be pretty confusing).